packages/core/src/awsService/appBuilder/serverlessLand/wizard.ts

```diff
@@ -0,0 +1,181 @@
+
+export interface CreateServerlessLandWizardForm {
+    name: string
+    location: vscode.Uri
+    pattern: string
+    runtime: string
+    iac: string
+}
+
+/**
+ * Wizard for creating Serverless Land applications
+ * Guides users through the project creation process
+ */
+export class CreateServerlessLandWizard extends Wizard<CreateServerlessLandWizardForm> {
+    private metadataManager: MetadataManager
+
+    public constructor(context: { defaultRegion?: string; credentials?: AWS.Credentials }) {
+        super({
+            exitPrompterProvider: createExitPrompter,
+        })
+        this.metadataManager = MetadataManager.getInstance()
+    }
+
+    public override async run(): Promise<CreateServerlessLandWizardForm | undefined> {
+        try {
+            // Load metadata from JSON file
+            const projectRoot = path.resolve(__dirname, '../../../../../')
+            const metadataPath = path.join(
+                projectRoot,
+                'src',
+                'awsService',
+                'appBuilder',
+                'serverlessLand',
+                'metadata.json'
+            )
+            await this.metadataManager.loadMetadata(metadataPath)
+
+            // Initialize and display pattern selection
+            const patterns = this.metadataManager.getPatterns()
+            if (patterns.length === 0) {
+                throw new Error('No patterns found in metadata')
+            }
+
+            const patternPicker = createQuickPick<string>(
+                patterns.map((p) => ({
+                    label: p.label,
+                    detail: p.description,
+                    data: p.label,
+                    buttons: [
+                        {
+                            iconPath: new vscode.ThemeIcon('github'),
+                            tooltip: 'Open in GitHub',
+                        },
+                        {
+                            iconPath: new vscode.ThemeIcon('open-preview'),
+                            tooltip: 'Open in Serverless Land',
+                        },
+                    ],
+                })),
+                {
+                    title: 'Select a Pattern for your application',
+                    placeholder: 'Choose a pattern for your project',
+                    buttons: createCommonButtons(),
+                    matchOnDescription: true,
+                    matchOnDetail: true,
+                }
+            )
+
+            const patternResult = await patternPicker.prompt()
+            if (!patternResult || typeof patternResult !== 'string') {
+                return undefined // User cancelled or invalid result
+            }
+            const selectedPattern = patternResult
+
+            // Show runtime options based on selected pattern
+            const runtimes = this.metadataManager.getRuntimes(selectedPattern)
+            if (runtimes.length === 0) {
+                throw new Error('No runtimes found for the selected pattern')
```

Comment:
same comment as above
```suggestion
                throw new ToolkitError('No runtimes found for the selected pattern')
```